packages/components/src/Stack/Stack.tsx

```diff
@@ -36,10 +42,16 @@ export const Stack = ({
   stretch = false,
   alignX,
   alignY,
+  asList = false,
   ...props
 }: StackProps) => {
+  const Component = asList ? 'ul' : 'div';
+  const StackChildren = asList
```

Comment:
lowercase, since it is not a component but a variable that holds a list of components